### DISCUSSION TITLE
Release 1.23.5 as Pre-release?

### DISCUSSION THREAD

**Discussion by madalinignisca** [cri-o/cri-o#6601] [General]
Hello. I'm confused why v1.23.5 is considered a pre-release. Isn't the z in x.y.z a patch for critical bug fixes or security issues? What would be the reason to tag it as a pre-release, which is expected on future new minors (y in x.y.z)?

**Comment by haircommander** [comment]
oops, that was my mistake. I've updated it

**Reply by madalinignisca** [reply]
many thanks.
